linode_api4/objects/vpc.py

```diff
@@ -24,25 +54,18 @@ class VPCSubnet(DerivedBase):
         "updated": Property(is_datetime=True),
     }
 
-    def _populate(self, json):
+    def _serialize(self):
         """
```

Comment:
I think we can keep the older `populate` method because it is functioning differently from `_serialize`.

Reply:
It looks like it's working without the populate method?